packages/styles/scss/components/tabs-extended/_tabs-extended.scss

```diff
@@ -206,6 +206,16 @@
       }
     }
   }
+  #{$prefix}--accordion__content {
```

Comment:
This hunk looks like an unintentional change that came in with commit https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/10840/commits/323c87167fc803cd51aa117b54e4cc4e89355dd1 . @marcelojcs was this intentional? 

If it is meant to be here, the opening and closing braces are unbalenced, which causes failures in the CI jobs.